std/concurrency.d

```diff
@@ -2473,6 +2474,23 @@ private
 
 version( unittest )
 {
+    void changeScheduler(Scheduler s)
+    {
+        import core.memory : GC;
+        import core.thread : Thread;
+
+        bool dosleep;
```

Comment:
I understand, I was suggesting that you changed the line to `bool doSleep = false;` (note capital `S`, in accordance with https://dlang.org/dstyle.html)